model_analyzer/result/result_manager.py

```diff
@@ -733,6 +742,26 @@ def _write_result(self,
                 table.to_formatted_string(separator=column_separator,
                                           ignore_widths=ignore_widths) + "\n\n")
 
+    def _add_default_to_results(self, model_name, results, result_heap):
+        '''
+        If default config is already in results, keep it there. Else, find and
+        add it from the result heap
+        '''
+        if not model_name:
+            return
+
+        found = False
+        for result in results:
+            if result.model_config().get_field(
+                    'name') == f"{model_name}_config_default":
+                found = True
```

Comment:
Needs a break here (like line 763) so that the remaining iterations of the loop are not executed

Reply:
I removed the breaks and just return early now